weaver/cli.py

```diff
@@ -0,0 +1,717 @@
+             "It will be inserted into an automatically generated request deploy body or into the provided one."
+    )
+    op_deploy_app_pkg.add_argument(
+        "--wps", dest="wps",
+        help="Reference URL to a specific process under a Web Processing Service (WPS) to package as OGC-API Process."
+    )
+    op_deploy_token = op_deploy.add_mutually_exclusive_group()
+    op_deploy_token.add_argument(
+        "-t", "--token", dest="token",
+        help="Authentication token to retrieve a Docker image reference from a private registry during execution."
+    )
+    op_deploy_creds = op_deploy_token.add_argument_group("Credentials")
+    op_deploy_creds.add_argument(
+        "-U", "--username", dest="username",
+        help="Username to compute the authentication token for Docker image retrieval from a private registry."
+    )
+    op_deploy_creds.add_argument(
+        "-P", "--password", dest="password",
+        help="Password to compute the authentication token for Docker image retrieval from a private registry."
+    )
+    op_deploy.add_argument(
+        "-D", "--delete", "--undeploy", dest="undeploy", action="store_true",
+        help="Perform undeploy step as applicable prior to deployment to avoid conflict with exiting process."
+    )
+
+    op_undeploy = ops_parsers.add_parser("undeploy", help="Undeploy an existing process.")
+    add_url_param(op_undeploy)
+    add_process_param(op_undeploy)
+
+    op_capabilities = ops_parsers.add_parser("capabilities", aliases=["processes"], help="List available processes.")
+    add_url_param(op_capabilities)
+
+    op_describe = ops_parsers.add_parser("describe", help="Obtain an existing process description.")
+    add_url_param(op_describe)
+    add_process_param(op_describe)
+
+    op_execute = ops_parsers.add_parser("execute", help="Submit a job execution for an existing process.")
+    add_url_param(op_execute)
+    add_process_param(op_execute)
+    # FIXME: support cumulative inputs for convenience? (ex: '-I input=value -I array=1,2 -I other=http://file')
+    op_execute.add_argument(
+        "-I", "--inputs", dest="inputs",
+        help="File path or URL reference to JSON or YAML contents defining job inputs with OGC-API or CWL schema."
+    )
+    # FIXME: support sync (https://github.com/crim-ca/weaver/issues/247)
+    # op_execute.add_argument(
+    #     "-A", "--async", dest="execute_async",
+    #     help=""
+    # )
+    op_execute.add_argument(
+        "-M", "--monitor", dest="monitor", action="store_true",
+        help="Automatically perform the monitoring operation following job submission to retrieve final results. "
+             "If not requested, the created job status location is directly returned."
+    )
+    add_timeout_param(op_execute)
+
+    op_dismiss = ops_parsers.add_parser(
+        "dismiss", help="Dismiss a pending or running job, or wipe any finished job results."
```

Comment:
dismiss is not implemented?
There is no corresponding function in the WeaverClient.

Reply:
oups
I'll add that
Great QA tester!

Reply:
Now available.